nxdrive/gui/view.py

```diff
@@ -398,7 +398,8 @@ def data(self, index: QModelIndex, role: int = REMOTE_PATH) -> Any:
         elif role == self.DESCRIPTION:
             description = str(row["description"])
             if not description:
-                description = "Session " + str(row["uid"])
+                uid = str(row["uid"])
+                description = f"Session {uid}"
```

Comment:
Not needed I think, you could simply type:
```suggestion
                description = f"Session {row['uid']}"
```
As f-string implicitely converts to `str`.